src/main/kotlin/apply/application/ApplicantAuthenticationService.kt

```diff
@@ -29,6 +31,13 @@ class ApplicantAuthenticationService(
         return jwtTokenProvider.createToken(applicant.email)
     }
 
+    fun generateAuthenticationCode(email: String): String {
+        applicantRepository.findByEmail(email)?.let { throw IllegalStateException() }
```

Comment:
이메일이 이미 등록되어 있는지를 메시지로 알려 주는 것이 좋을지는 고민이지만, `check()`를 사용하는 것이 코드를 이해하기에 더 좋은 것 같아요.

```suggestion
        check(!applicantRepository.existsByEmail(email)) { "이미 등록된 이메일입니다." }
```

Reply:
반영하였습니다:)

Reply:
쿼리 메서드가 아닌 `@Query`를 사용한 이유가 궁금해요!

Reply:
다른 레포짓토리 메소드를 따라가기 위해서 `@Query` 를 사용하였는데, `existsByInformationEmail` 로 사용하는 것이 더 좋다고 생각하시는 걸까요? 👀